datapipelines/src/test/java/io/customer/datapipelines/DataPipelinesCompatibilityTests.kt

```diff
@@ -432,6 +433,37 @@ class DataPipelinesCompatibilityTests : JUnitTest() {
         payloadContext.containsKey("platform") shouldBeEqualTo false
     }
 
+    @Test
+    fun device_givenTokenRegisteredWithNullableDeviceAttributes_expectDeviceAttributesAreSent() = runTest {
+        val givenToken = String.random
+        val givenAttributes = mapOf(
+            "cio_sdk_version" to "2.0.0-alpha.10",
+            "app_release" to "1.0",
+            "app_build" to "100",
+            "app_namespace" to "io.customer.android",
+            "device_os_version" to "13",
```

Comment:
Should be int to match with `DeviceStoreImpl`?

```suggestion
            "device_os_version" to 13,
```

Reply:
`device_os_version ` is just a random key :D we take that value in `device_os` i'll update